sunpy/net/helioviewer.py

```diff
@@ -189,6 +192,57 @@ def download_jp2(self, date, observatory=None, instrument=None, detector=None,
 
         return self._get_file(params, directory=directory, overwrite=overwrite)
 
+    def get_jp2_header(self, date, observatory=None, instrument=None, detector=None,
+                       measurement=None, jp2_id=None):
+        """
+        Get the XML header embedded in a JPEG2000 image. Includes the FITS header as well as a section 
+        of Helioviewer-specific metadata.
+
+        This uses `getJP2Header <https://api.helioviewer.org/docs/v2/#JPEG2000>`_ from the Helioviewer API.
+
+        Parameters
+        ----------
+        observatory : `str`
+            Observatory name
+        instrument : `str`
+            Instrument name
+        measurement : `str`
+            Measurement name
+        detector : `str`
+            Detector name
+
+        Returns
+        -------
+        out : `dict`
+            Returns a dictionary containing the header information of JPEG 2000 image.
+
+        Examples
+        --------
+        >>> from sunpy.net import helioviewer
+        >>> hv = helioviewer.HelioviewerClient()  # doctest: +REMOTE_DATA
+        >>> header = hv.get_jp2_header('2012/07/03', observatory='SDO',
+        ...                            instrument='HMI', detector=None, measurement='continuum')  # doctest: +REMOTE_DATA
+        >>> fits_header = header['meta']['fits']  #The keys 'meta' and 'fits' can be used to get the fits header information
+        >>> helioviewer_meta_data = header['meta']['helioviewer'] #The keys 'meta' and 'helioviewer' can be used to extract the
+                                                                  #helioviewer specific metadata.
```

Comment:
These comments need to be moved. They should be above the line of code, otherwise the line is too long. 

Also these will need `# doctest: +REMOTE_DATA`